ci-operator/config/openshift/release/openshift-release-master__ci-4.19.yaml

```diff
@@ -82,6 +82,19 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-ovn-serial
+- as: e2e-aws-ovn-serial-tls-13
+  interval: 168h
+  steps:
+    cluster_profile: aws-2
+    env:
+      BASE_DOMAIN: aws-2.ci.openshift.org
+      TEST_SUITE: openshift/conformance/serial
```

Comment:
i'm not sure which suite is the best, but the serial one might take a long time to execute.

Reply:
Yeah we shouldn't need this to be serial based on my current understanding anymore, changed